MAUI/Autocomplete/Selection.md

```diff
@@ -78,12 +78,76 @@ The following gif image illustrates the result of the above code:
 
 ![.NET MAUI Autocomplete with single selection mode](Images/Selection/SingleSelection.png)
 
+## Multiple selection
+
+The AutoComplete allows user to select multiple values by beginning to enter the input and selecting items from the drop-down list. The multi-select AutoComplete mode can be enabled by setting the `SelectionMode` property as `Multiple`. 
```

Comment:
The AutoComplete allows the user to select multiple values by beginning to enter the input and selecting items from the drop-down list. The multi-select AutoComplete mode can be enabled by setting the `SelectionMode` property as `Multiple`.